### PR TITLE
Issue#832

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding an important note for #832

### CODE REVIEW

**Comment by markjjo** [comment]
@j0rt3g4 Please re-write the note that you added to audit log topic. "Privileges" is misspelled twice, and I'm not really sure what the note is trying to convey. Instead of focusing on admin "privileges", you might consider focusing on Exchange admin tasks. Additionally, I would move this note to the section on the Exchange admin audit log (https://docs.microsoft.com/en-us/office365/securitycompliance/search-the-audit-log-in-security-and-compliance#exchange-admin-audit-log) where we already discuss Search-UnifiedAuditLog vs. Search-AdminAuditLog

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/search-the-audit-log-in-security-and-compliance.md [hunk 1]
```diff
@@ -782,6 +785,13 @@ Here are some tips for searching for Exchange admin activities when searching th
   
 - To get information about what cmdlet was run, which parameters and parameter values were used, and what objects were affected, you can export the search results by selecting the **Download all results** option. For more information, see [Export, configure, and view audit log records](export-view-audit-log-records.md). 
     
+
+- You can also view events in the Exchange admin audit log by using the Exchange admin center. For instructions, see [View the administrator audit log](https://technet.microsoft.com/library/dn342832%28v=exchg.150%29.aspx).
```
Comment (JohanFreelancer9) [inline SecurityCompliance/search-the-audit-log-in-security-and-compliance.md]:
```suggestion
- You can also view events in the Exchange admin audit log by using the Exchange admin center. For more information, see [View the administrator audit log](https://technet.microsoft.com/library/dn342832%28v=exchg.150%29.aspx).
```

File: SecurityCompliance/search-the-audit-log-in-security-and-compliance.md [hunk 2]
```diff
@@ -782,6 +785,13 @@ Here are some tips for searching for Exchange admin activities when searching th
...
+
+> [!IMPORTANT]
+>  To search and track the use of exchange online administrator privilegies use the Cmdlet *Search-AdminAuditLog* over the *Search-UnifiedAuditLog*.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/search-the-audit-log-in-security-and-compliance.md]:
```suggestion
>  To search and track the use of exchange online administrator privileges, use the Cmdlet *Search-AdminAuditLog* over the *Search-UnifiedAuditLog*.
```

File: SecurityCompliance/threat-explorer.md [hunk 3]
```diff
@@ -165,3 +165,8 @@ To learn more about roles and permissions, see the following resources:
 - [Permissions in the Office 365 Security &amp; Compliance Center](permissions-in-the-security-and-compliance-center.md)
 - [Feature permissions in Exchange Online](https://docs.microsoft.com/exchange/permissions-exo/feature-permissions)
   
+## Some differences between Real time detection and Threat Exporter.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/threat-explorer.md]:
```suggestion
## Some differences between real-time detection and Threat Explorer
```

File: SecurityCompliance/threat-explorer.md [hunk 4]
```diff
@@ -165,3 +165,8 @@ To learn more about roles and permissions, see the following resources:
...
+
+ - **Real-Time detection report** is shipped with ATP P1 subscription while **Threat Explorer** comes with ATP P2.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/threat-explorer.md]:
```suggestion
 - **Real-time detection** is shipped with ATP Plan 1 subscriptions while **Threat Explorer** comes with ATP Plan 2.
```

File: SecurityCompliance/threat-explorer.md [hunk 5]
```diff
@@ -165,3 +165,8 @@ To learn more about roles and permissions, see the following resources:
...
+ - **Real-Time detections** report allows you to show up the detections in real-time while threat explorer reports allow you to see more details in a given attack.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/threat-explorer.md]:
```suggestion
 - ****Real-time detection** allows you to view the detections in real-time while threat explorer reports allow you to see more details in a given attack.
```

File: SecurityCompliance/threat-explorer.md [hunk 6]
```diff
@@ -165,3 +165,8 @@ To learn more about roles and permissions, see the following resources:
...
+ - **Threat Explorer** reports offer different sets of reports that enhance your experience respect **Real-Time Detector** reports.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/threat-explorer.md]:
```suggestion
 - **Threat Explorer** reports offer different sets of reports that enhance your experience in relation to **real-time detection** reports.
```

**Pull request merged**